services/graph/pkg/config/http.go

```diff
@@ -2,7 +2,7 @@ package config
 
 // HTTP defines the available http configuration.
 type HTTP struct {
-	Addr      string `yaml:"addr" env:"GRAPH_HTTP_ADDR"`
+	Addr      string `yaml:"addr" env:"GRAPH_HTTP_ADDR" desc:"The bind address of the HTTP service."`
 	Namespace string `yaml:"-"`
-	Root      string `yaml:"root" env:"GRAPH_HTTP_ROOT"`
+	Root      string `yaml:"root" env:"GRAPH_HTTP_ROOT" desc:"The root path of the HTTP service."`
```

Comment:
The default is: `/graph`. In which folder can /graph be found?

Reply:
Clarified with @dragonchaser

Reply:
I changed it to `Subdirectory that serves as the root for this HTTP service.` for all services that have that option.

if you have oCIS on `https://ocis.owncloud.test`, the graph service is available on `https://ocis.owncloud.test/graph`. If you want to have it on another subdirectory, eg. `/foobar`, you could change that config and also the proxy route to serve the graph on `https://ocis.owncloud.test/foobar`